File: Assignment 4/Activity 3.js
function main() {
    // Create a program that asks the user for a distance in miles, and then calculate and display the distance in yards, feet, and inches, or ask the user for a distance in miles, and then calculate and display the distance in kilometers, meters, and centimeters.
    var mile;
    var distanceFeet;
    var distanceYard;
    var distanceInches;
    var distanceKilometer;
    var distanceMeter;
    var distanceCentimeter;
    
    window.alert(" What is the Distance in Miles ? ");
    mile = window.prompt('Enter a value for mile');
    distanceFeet = mile * 5280;
    distanceYard = mile * 1760;
    distanceInches = mile * 63360;
    distanceKilometer = mile * 1.6069344;
    distanceMeter = mile * 1609.34;
    distanceCentimeter = mile * 160934;
    window.alert(" The distance in Feet is = " + distanceFeet);
    window.alert(" The distance in Yard is = " + distanceYard);
    window.alert(" The distance in Inches is = " + distanceInches);
    window.alert(" The distance in kilometer is = " + distanceKilometer);
    window.alert(" The distance in Meter is = " + distanceMeter);
    window.alert(" The distance in centimeter is = " + distanceCentimeter);
}

File: Assignment 4/Activity 3.py
# Distance Converter

miles = int(input("What is the distance in miles?"))

DISTANCE_IN_FEET = miles * 5280
DISTANCE_IN_YARD = miles * 1760
DISTANCE_IN_INCHES = miles * 63360

print("The distance in feet is = " + str(DISTANCE_IN_FEET))
print("The distance in yard is = " + str(DISTANCE_IN_YARD))
print("The distance in inches is = " + str(DISTANCE_IN_INCHES))

